packages/core/src/components/selectable/selectable.scss

```diff
@@ -3,9 +3,10 @@
 @use "~dso-toolkit/src/variables/units";
 
 :host {
+  --selectable-indent: #{units.$u4};
```

Comment:
Waarom een custom property?

Reply:
Omdat ik 'm 4 keer nodig heb. Als ooit aan die indent gesleuteld moet worden, hoeft dat maar 1 keer

Reply:
Maar dit kan toch ook met Sass variables?

Reply:
Ja, maar als die `$u4` een `$u5` moet worden..
Natuurlijk kan je ook een aparte Sass variable aanmaken, maar een CP is hier ook geknipt voor; sterker nog, daar zijn ze min of meer voor uitgevonden: 
1) voor de efficiency
2) te cachen
3) voor de semantiek; hier is het een indent (om iets in te plaatsen).
4 keer '32px' is een stuk minder duidelijk qua wat de auteur hiermee wil ;)

Reply:
> Ja, maar als die `$u4` een `$u5` moet worden.. Natuurlijk kan je ook een aparte Sass variable aanmaken, maar een CP is hier ook geknipt voor; sterker nog, daar zijn ze min of meer voor uitgevonden:
> 
> 1. voor de efficiency
> 2. te cachen
> 3. voor de semantiek; hier is het een indent (om iets in te plaatsen).
>    4 keer '32px' is een stuk minder duidelijk qua wat de auteur hiermee wil ;)

Mee eens en ik ben blij dat we vanuit _named values_ denken :) Maar waar ik voor wil waken is dat we straks een mix van Sass variables en Custom Properties hebben, die hetzelfde doel hebben.

We hebben het hier een tijd terug ook over gehad, en volgens mij kwamen we tot de volgende conclusie:
* Het is niet de moeite om alle Sass variables te herschrijven tot custom properties. Sass is here to stay.
* Daarmee blijven Sass variables de basis en blijven we zoveel mogelijk compiletime genereren.
* Uitzonderingen zijn: 
  * interactie/gedrag tussen componenten*, bijvoorbeeld component A in B krijgt een andere achtergrondkleur.
  * Responsive gedrag: Geen selectors meer overschrijven maar alleen de waardes die worden gezet

Met bovenstaand in het achterhoofd begrijp ik de voorgestelde wijziging niet, en is het wellicht een idee om dit soort conclusies/afspraken formeler vast te leggen.

We kunnen het hier natuurlijk nog steeds over hebben, maar laten we dat niet in het kader van dit issue doen.

_*) Custom properties zijn geen onderdeel van de public API van de toolkit en zijn daarmee allemaal private en zijn dus geen factor bij het bepalen van een breaking/feature/patch release._

Reply:
Sja, dat druist een beetje in tegen hoe ik gewend ben met CP's om te gaan: normaal gesproken plemp ik er een heleboel in de `:root` en gebruik die overal en nergens in de hele codebase.

aan de andere kant begrijp ik jouw argument ook, dus bij deze omgezet ;)

Reply:
Dank!